test/sanity/validate-modules/main.py

```diff
@@ -988,8 +989,8 @@ def _check_for_new_args(self, doc):
 
         with CaptureStd():
             try:
-                existing_doc, _, _, _ = get_docstring(self.base_module, verbose=True)
-                existing_options = existing_doc.get('options', {}) or {}
```

Comment:
The `or` here was recently added in https://github.com/ansible/ansible/commit/90ac38bd4c7d0de73ec9d66fe8c4e8424a027982 Isn't there still a case on a new module submission where `options` is `None` and has no fragment?

Reply:
Thanks for catching that, looks like bcoca has since modified that part (https://github.com/ansible/ansible/pull/31677/commits/2be4d98a35f333cd82102cd362fc8b2efeeb39f2#diff-5fcdebb0855843886d6ee47612aa48d3L991) too. I'll drop that commit and cp his latest.